app/views_redesigned/properties/_standard_card.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= card do %>
+  <%= card_header do %>
+
+    <%= card_title add_class: "mb-0" do %>
+      <div class="d-flex justify-content-between align-items-center">
+        <h6 class="mb-0">
+          <%= tag.span "", class: "fas fa-circle small mr-1 text-#{property_status_color(property.status)}",
+            title: property_status_title(property.status), data: tooltip_expando %>
+          <%= link_to property.name, property, class: "link-muted" %>
+        </h6>
+        <div class="position-relative">
+          <%= link_to property_property_screenshots_path(property), method: :put, class: "btn btn-xs btn-icon btn-light transition-3d-hover" do %>
+            <span class="fas fa-camera btn-icon__inner"></span>
+          <% end if authorized_user.can_admin_system? %>
+          <%= link_to property.url, class: "btn btn-xs btn-icon btn-light transition-3d-hover", target: "_blank" do %>
```

Comment:
`transition-3d-hover` is not a class in our new theme.

Reply:
Good catch!